integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -831,4 +832,12 @@ object CommonUtil {
         LOGGER.error(s)
     }
   }
+
+  def getScaleAndPrecision(dataType: String): (Int, Int) = {
```

Comment:
This method is duplicated. Please reuse it

Reply:
ok

Reply:
Moved this method to commonutils and updated all the callers